backend/src/test/java/heartbeat/service/jira/JiraBoardConfigDTOFixture.java

```diff
@@ -224,6 +235,54 @@ public static JiraBoardSetting.JiraBoardSettingBuilder JIRA_BOARD_SETTING_BUILD(
 					TargetField.builder().key("testKey3").name("Flagged").flag(true).build()));
 	}
 
+	public static JiraBoardSetting.JiraBoardSettingBuilder CLASSIC_JIRA_BOARD_SETTING_BUILD() {
+		return JiraBoardSetting.builder()
+			.boardId("unknown")
+			.boardColumns(List.of(RequestJiraBoardColumnSetting.builder().name("In Dev").value("In Dev").build(),
+					RequestJiraBoardColumnSetting.builder()
+						.name("Waiting for testing")
+						.value("Waiting for testing")
+						.build(),
+					RequestJiraBoardColumnSetting.builder().name("Block").value("Block").build(),
```

Comment:
refactor like `"Block"` as a constant in the fixture, not using string everywhere

Reply:
extract constant in the fixture!